python-frontend/src/main/java/org/sonar/python/semantic/v2/typeshed/TypeShedDescriptorsProvider.java

```diff
@@ -128,4 +139,35 @@ static ModuleSymbol deserializedModule(String moduleName, InputStream resource)
     }
   }
 
+
+  /**
+   * Returns stub symbols to be used by SonarSecurity.
+   * Ambiguous symbols that only contain class symbols are disambiguated with latest Python version.
+   */
+  public Collection<Symbol> stubFilesSymbols(ProjectLevelSymbolTable projectLevelSymbolTable) {
```

Comment:
I think that the `Descriptors` provider should not operate with the `Symbol` model, it should just provide descriptors. The conversion from the descriptor to the symbol model is on the `projectLevelSymbolTable` side